source/Sylvan.Data.Excel/Xlsx/XlsxWorkbookReader.cs

```diff
@@ -155,10 +155,8 @@ public XlsxWorkbookReader(Stream iStream, ExcelDataReaderOptions opts) : base(iS
 					for (int idx = 0; idx < xfMap.Length; idx++)
 					{
 						var xf = (XmlElement)xfsElem.ChildNodes[idx];
-						if (xf.HasAttribute("applyNumberFormat"))
-						{
-							xfMap[idx] = int.Parse(xf.GetAttribute("numFmtId"));
-						}
+						var fmtIdStr = xf.GetAttribute("numFmtId");
```

Comment:
@MarkPflug 
fmtIdStr may be null:
![image](https://github.com/MarkPflug/Sylvan.Data.Excel/assets/130081973/8412e155-ebcb-4cb3-951c-e2394197ddfa)